backend/theme/admin.py

```diff
@@ -88,13 +96,13 @@ def body_font_preview(self, obj):
         return "No font selected"
 
     def logo_preview(self, obj):
-        if obj.logo_url:
-            return mark_safe(f'<img src="{obj.logo_url}" style="max-height: 50px;"/>')
+        if obj.logo_image:
+            return mark_safe(f'<img src="/upload/{obj.logo_image.file}" style="max-height: 50px;"/>')
```

Comment:
I'm not sure if this is going to work with a variable server subpath like `/server/`

Reply:
For the admin interface, this works like a charm.

Reply:
But also on ACC & TST, where the path might be `/server/upload`?

Reply:
Will check!